src/stack-hci/azext_stack_hci/generated/utils.py

```diff
@@ -0,0 +1,28 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from importlib import import_module
+
+'''
+Since the method get_models()/get_sdk() in azure-cli-core is bound to resourceType, the method _sdk_get_versioned_sdk() will be called, so only the multi-api SDK is supported.
+If we add a version directory to the path of the SDK folder, because there is no required client (specific to the multiple version SDK) outside of the version directory, the client_factory will have a load problem.
+So write a separate method to load the model.
+'''
+
+
+def get_model(sdk_path, mod_attr_path, checked=True):
```

Comment:
seems we could import model directly. If yes, no need to add this function

Reply:
Yes, I'll import it directly~